tests/ecosystem/deployment/test_deployment.py

```diff
@@ -39,3 +45,40 @@ def test_deployment(pvc_factory, pod_factory):
         log.info(
             "Cluster will be destroyed during teardown part of this test."
         )
+
+
+@skipif_no_encrypted
+@deployment
+def test_verify_osd_encryption():
+    """
+    Verify the OSD is encrypted
+    """
+    ocs_version = float(config.ENV_DATA['ocs_version'])
+    if ocs_version < 4.6:
+        error_message = "Encryption at REST can be enabled only on OCS >= 4.6!"
+        raise UnsupportedFeatureError(error_message)
+
+    osd_node_names = get_osd_running_nodes()
+    osd_size = get_osd_size()
+    lsblk_output_list = []
+    for worker_node in osd_node_names:
+        lsblk_cmd = 'oc debug node/' + worker_node + ' -- chroot /host lsblk'
+        log.info(lsblk_cmd)
```

Comment:
this logging is not needed because `out = run_cmd(lsblk_cmd)` will print the executed command